INTRODUCTION.md

```diff
@@ -701,15 +701,106 @@ which returns one of the following values:
 This method should be called by the application on delivery report error.
 
 
+### Transactional Producer
 
+#### FIXME: misc
+
+To make sure messages time out (in case of connectivity problems, etc) within
+the transaction, the `message.timeout.ms` configuration property must be
+set lower than the `transaction.timeout.ms`.
+If `message.timeout.ms` is not explicitly configured it will be adjusted
+automatically.
```

Comment:
I would add the text "(this is enforced)" after "set lower than the `transaction.timeout.ms`" - i assume it is. it's nice to know from the documentation that this is not something I need to worry about (i.e. if i screw up the config i can be sure to get an error).